packages/duf/src/evo/data_converters/duf/importer/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .duf_lineset_to_evo import convert_duf_lineset
```

Comment:
Every file you add (excluding the data files, I guess) will need to have the copy pasted license header block.